install/kubernetes/cilium/values.yaml

```diff
@@ -3053,6 +3053,12 @@ operator:
     # -- Interval, in seconds, to check if there are any pods that are not
     # managed by Cilium.
     intervalSeconds: 15
+    # @schema
+    # type: [null, string]
+    # @schema
+    # -- Selector for pods that should be restarted when not managed by Cilium.
+    # If not set, defaults to built-in selector "k8s-app=kube-dns". Set to empty string to select all pods.
+    selector: ~
```

Comment:
Please apply this change to `values.yaml.tmpl` - as the `values.yaml` file is auto-generated